go.mod

```diff
@@ -171,6 +170,7 @@ replace cosmossdk.io/x/protocolpool => ./x/protocolpool
 
 // Below are the long-lived replace of the Cosmos SDK
 replace (
+	cosmossdk.io/x/group => ../x/group
```

Comment:
We should strive to have the SDK not importing group. Is that possible? We want to avoid circular deps.

Reply:
It is a replace directive that I planned on removing once all was working right, but you are correct, thanks @julienrbrt!

Reply:
Ah, sorry, I saw the PR was R4R that's why I had a look 😃

Reply:
Not a problem and thank you @julienrbrt!